examples/custom-james-assembly/custom-james-assembly.iml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please configure a local gitignore onto your laptop so that you do not commit ide file into an Apache project

(without modifying the project gitignore)